### DISCUSSION TITLE
Popover Token

### DISCUSSION THREAD

**Discussion by callum-rosel** [cloudscape-design/components#956] [Q&A] [answered]
Hi,

Looking at 'theming' and I cannot seem to find a design token for popovers? 

Searching for $color-background-popover yields no result.

Is there another way i can achieve this?

I tried adding a container for the content of the popover, but it still maintains a region around the container that is the same color of the popover.

**Comment by rubencarvalho** [comment, marked as answer]
Hello,

The popover token you refer to was indeed, not available as a themeable token. I've went ahead an opened a [PR](https://github.com/cloudscape-design/components/pull/1096) to address your feedback.

We'll update this discussion once the change has been released.

Thank you!
